internal/fingerprint/fingerprint.go

```diff
@@ -43,6 +43,8 @@ var EXCLUDED_FILES = []string{
 	"thumbs.db", "babel.config.js", "license.txt", "license.md", "copying.lib", "makefile",
 }
 
+var FILES_TO_UNPACK = []string{".jar", ".nupkg", ".war"}
```

Comment:
can be a const I think?

Reply:
Const only works for scalar values, not arrays, slices etc.